rust-src/concordium_base/src/web3id/v1/proofs.rs

```diff
@@ -0,0 +1,1128 @@
+use crate::random_oracle::StructuredDigest;
+use crate::web3id::{LinkingProof, PresentationVerificationError, ProofError};
+use crate::{
+    curve_arithmetic::Curve,
+    id::types::{Attribute, GlobalContext},
+    random_oracle::RandomOracle,
+};
```

Comment:
```suggestion
use crate::web3id::{LinkingProof, PresentationVerificationError, ProofError};
use crate::{
    curve_arithmetic::Curve,
    id::types::{Attribute, GlobalContext},
    random_oracle::{StructuredDigest,RandomOracle},
};
```

Similar the imports to `use crate::id:: ... ` could be consolidated.